### PR TITLE
chore: bump version amplify_flutter 0.3.0.rc.4 and amplify_authenticator 0.1.0.rc.3

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
*Issue #, if available:* https://github.com/aws-amplify/amplify-flutter/issues/1258

*Description of changes:*
- bump version of amplify to 0.3.0.rc.4
- bump version of amplify_authenticator 0.1.0.rc.3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_authenticator/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 0.1.0-rc.3 (2022-01-12)
```
Comment (haverchuck) [inline packages/amplify_authenticator/CHANGELOG.md:1]: Shouldn't we have something in the changelog here?

**Review by haverchuck — COMMENTED** [review]
I think we need some content in the changelog for amplify_authenticator - and in some other packages as well.

File: packages/amplify_authenticator/CHANGELOG.md (hunk 1 above)
Comment (Jordan-Nelson) [reply packages/amplify_authenticator/CHANGELOG.md:1]: The only change is that the authenticator not requires a higher version of amplify_flutter.

**Review by haverchuck — APPROVED** [review]

File: packages/amplify_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+## 0.3.0-rc.4 (2022-01-12)
```
Comment (HuiSF) [inline packages/amplify_flutter/CHANGELOG.md:1]: Should we announce the breaking change about
```
import 'package:amplify_flutter/amplify.dart';
to 
import 'package:amplify_flutter/amplify_flutter.dart';
```
here and probably a migration instruction?

Comment (Jordan-Nelson) [reply packages/amplify_flutter/CHANGELOG.md:1]: I will add this along with change logs for the rest of the packages

File: packages/amplify_authenticator/CHANGELOG.md (hunk 1 above)
Comment (Jordan-Nelson) [reply packages/amplify_authenticator/CHANGELOG.md:1]: Updated

File: packages/amplify_flutter/CHANGELOG.md (hunk 2 above)
Comment (Jordan-Nelson) [reply packages/amplify_flutter/CHANGELOG.md:1]: Updated

**Review by dnys1 — APPROVED** [review]

**Pull request merged**